lib/main/STM32H7/Drivers/CMSIS/Device/ST/STM32H7xx/Include/stm32h750xx.h

```diff
@@ -7,7 +7,7 @@
   *          This file contains:
   *           - Data structures and the address mapping for all peripherals
   *           - Peripheral's registers declarations and bits definition
-  *           - Macros to access peripheral�s registers hardware
+  *           - Macros to access peripheral�s registers hardware
```

Comment:
Are you using a non-unicode compliant text editor?

Reply:
I must have fiddled with a rollover key. Is this the only occurrence?

Reply:
Yeah, only occurrence.